sys/shell/commands/sc_netif.c

```diff
@@ -120,15 +120,15 @@ void _netif_send(int argc, char **argv)
     ng_netapi_send(dev, pkt);
 }
 
-void _netif_addr(int argc, char **argv)
+void _netif_addr_get(int argc, char **argv)
 {
     kernel_pid_t dev;
     uint8_t addr[MAX_ADDR_LEN];
     size_t addr_len;
     int res;
 
-    if (argc < 2) {
-        printf("usage: %s IF [ADDR]\n", argv[0]);
+    if (argc < 3) {
+        printf("usage: %s IF ADDR_LEN (in byte)\n", argv[0]);
```

Comment:
This parameter is basically used to distinguish between different address types (short/long). If the user does not specify any length, there would be no way in knowing which address the user wants to know...